manager/cve_handler.py

```diff
@@ -89,6 +89,7 @@ def _full_query(rh_account_id, synopsis):
                        SystemPlatform.last_evaluation,
                        SystemPlatform.advisor_evaluated.alias('rules_evaluation'),
                        SystemPlatform.last_upload,
+                       SystemPlatform.culled_timestamp,
```

Comment:
there isn't stale_timestamp and stale_warning_timestamp returned in this endpoint, it doesn't make sense to return culled_timestamp alone

Reply:
Do we need to return `stale_timestapm` and `stale_warning_timestamp` also? Or just not return `culled_timestamp` here?